Jenkinsfile

```diff
@@ -1,4 +1,4 @@
-@Library('jenkins-library') _
+@Library('jenkins-library@feature/DOPS-2746/rust_sonar_dojo_slither') _
```

Comment:
Forgot?

Reply:
Nope, I leave it because we did not update defectDojo to support slither reports. 
Now it is. Removed